app/models/api/APIModel.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.api
+
+import play.api.libs.json.{Json, OFormat}
+
+case class APIModel(
+                     noOfPoints:Int
+                   )
+
+object APIModel {
+  implicit val format: OFormat[APIModel] = Json.format[APIModel]
```

Comment:
Could you replace OFormat with Writes
```suggestion
    implicit val format: Writes[APIModel] = Json.writes[APIModel]
```